pypiserver/bottle.py

```diff
@@ -89,17 +89,15 @@ def _cli_patch(args):
 
             def json_dumps(data):
                 raise ImportError(
-                    "JSON support requires Python 2.6 or simplejson.")
+                    "JSON support requires simplejson.")
```

Comment:
I would just not touch the `bottle.py` - that's an "upstream-derived" file for *pypiserver*, we don't maintain it.

I know that your changes are not invasive and make sanse, but 
there is another reason to keep this file pristine: 
when we upgrade *bottle*, [sometimes](https://github.com/pypiserver/pypiserver/commit/2d3f7bf) certain changes are needed to fix things.  Leaving `bottle.py` untouched makes these cases stand out, and be easier to redo those changes later, on each new upgrade.

Reply:
Makes sense, I'll revert those.